repository/src/main/java/org/alfresco/repo/security/authentication/identityservice/IdentityServiceConfig.java

```diff
@@ -69,6 +69,13 @@ public class IdentityServiceConfig
     private boolean clientIdValidationDisabled;
     private String adminConsoleRedirectPath;
     private String signatureAlgorithms;
+    private String adminConsoleScopes;

```

Comment:
Maybe this is a good oportunity to convert this class to `record` and inject properties by constructor injection?

Reply:
I would avoid converting `IdentityServiceConfig` class to `record` because:
1. It would require changes in many references to that class.
2. This change should be implemented in a way that it’s backportable to the ACS 23.4. Converting to `record` would require new compatible version of the AOS amp due to the references to the `IdentityServiceConfig` class. (See: https://github.com/Alfresco/alfresco-aos-module/blob/support/SP/3.0.N/alfresco-aos-repo-binding/src/main/java/org/alfresco/module/aosmodule/auth/service/AosAuthenticationUriService.java#L86)